cmd/theatre-consoles/main.go

```diff
@@ -87,7 +94,13 @@ func Run(ctx context.Context, logger kitlog.Logger) error {
 	// Match on the kingpin command and enter the main command
 	switch cmd {
 	case create.FullCommand():
-		return Create(ctx, logger, runner, namespace)
+		return Create(
+			ctx, logger, runner,
+			CreateOptions{
+				Namespace: namespace,
```

Comment:
Just use *cliNamespace here?

Reply:
Given that if we inline the newRunner() we probably wont have the assignment lines anymore

```
// Get context and namespace from flags
kctx := *cliContext
namespace := *cliNamespace
```